README.md

```diff
@@ -80,28 +77,38 @@ When you run the sample, if you are running on a domain joined or AAD joined Win
-1. On the top bar, click on your account, and then on **Switch Directory**.
-1. Once the *Directory + subscription* pane opens, choose the Active Directory tenant where you wish to register your application, from the *Favorites* or *All Directories* list.
-1. Click on **All services** in the left-hand nav, and choose **Azure Active Directory**.
-
-> In the next steps, you might need the tenant name (or directory name) or the tenant ID (or directory ID). These are presented in the **Properties**
-of the Azure Active Directory window respectively as *Name* and *Directory ID*
-
-#### Register the client app (active-directory-dotnet-iwa)
-
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
-1. Enter a friendly name for the application, for example 'active-directory-dotnet-iwa' and select 'Native' as the *Application Type*.
-1. For the *Redirect URI*, enter `https://<your_tenant_name>/active-directory-dotnetcore-iwa`, replacing `<your_tenant_name>` with the name of your Azure AD tenant.
-1. Click **Create** to create the application.
-1. In the succeeding page, Find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. Then click on **Settings**, and choose **Properties**.
-1. Configure Permissions for your application. To that extent, in the Settings menu, choose the 'Required permissions' section and then,
-   click on **Add**, then **Select an API**, and type `Microsoft Graph` in the textbox. Then, click on  **Select Permissions** and select **User.ReadBasic.All**.
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account or a personal Microsoft account.
+1. If your account gives you access to more than one tenant, select your account in the top right corner, and set your portal session to the desired Azure AD tenant
+   (using **Switch Directory**).
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview)**.
+
+#### Register the client app (iwa-console)
+
+1. In **App registrations (Preview)** page, select **New registration**.
+1. When the **Register an application page** appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `iwa-console`.
+   - In the **Supported account types** section, select **Accounts in any organizational directory**.
+   - Select **Register** to create the application.
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. In the list of pages for the app, select **Manifest**, and:
+   - In the manifest editor, set the ``allowPublicClient`` property to **true** 
```

Comment:
humm - do you really need to do this? Would this not work if user selects - for example, any of the Suggested REdirect URIs for public clients instead?

Reply:
no, it does not work with one of the pre-suggested URLs (and this is not a really good experience imho). IWA uses either federation or ROPC (depending on if the user is managed or federated), and therefore this is the same case as ROPC.
Maybe we should have a check-box: "use ROPC" ? like with have for Implicit flow? the experience would be better?